packages/openapi3/test/tsp-openapi3/convert-openapi3-doc.test.ts

```diff
@@ -0,0 +1,49 @@
+import { formatTypeSpec } from "@typespec/compiler";
+import { strictEqual } from "node:assert";
+import { describe, it } from "vitest";
+import { convertOpenAPI3Document } from "../../src/index.js";
+
+describe("tsp-openapi: convertOpenAPI3Document", () => {
```

Comment:
you don't need the top level describe, we used to have those for mocha as it flattened everything and made it impossible to find but vitest groups per file already so it adds extra nesting

Reply:
Sorry - I keep doing that out of habit!

Reply:
tbh it could also be worth a discussion as not having it has the inconvenience of not being able to run all the test in a file without using the explorer. I think its worht not having an extra nesting just for that but this has been a caveat of doing it

Reply:
For what it’s worth, I haven’t really had any issues running tests for my developer inner loop elsewhere where it is flattened. Easy to run individual tests in vscode, and use the cli for verifying what I broke.